scripts/generate-chapter-reading-homework-pdfs.ts

```diff
@@ -0,0 +1,97 @@
+import { readFile, writeFile, mkdir } from 'fs/promises';
+import { ExerciseData } from 'src/types';
+import util from 'util';
+const exec = util.promisify(require('child_process').exec);
+import path from 'path';
+
+const filterName = process.argv[3];
+
+const readExercises = async () => {
+  const exercisesJsonPath = process.argv[2];
+  const fileContents = await readFile(exercisesJsonPath, 'utf8');
+  return JSON.parse(fileContents).exercises;
+};
+
+const filterAndGroupExercises = (exercises: ExerciseData[]) => {
+  const moduleGroups = {};
+  let filteredExercises = exercises;
+
+  if (filterName === 'single-part-multiple-choice') {
+    console.log(`using filter ${filterName}`);
```

Comment:
maybe this was meant to go above the conditional?

Reply:
This was more of a paranoid check so I can see it if it's running the filter... I may have run it once or twice with a typo 😆  I should probably add validation the beginning of the script if we add other filters 🤔